redisgraph/execution_plan.py

```diff
@@ -0,0 +1,35 @@
+class ExecutionPlan:
+    def __init__(self, plan):
+        self.plan = plan
+
+    def __str__(self) -> str:
+        return "\n".join(self.plan)
+
+    def operation_tree(self):
```

Comment:
It seems to me like this function needs to be "private" and should be called by `__init__`